src/topology/uniform_space/cauchy.lean

```diff
@@ -577,7 +577,7 @@ indefinite_description _ $ (cauchy_iff.1 hf).2 (U n) (U_mem n)
 
 /-- Given a Cauchy filter `f` and a sequence `U` of entourages, `set_seq` provides
 an antitone sequence of sets `s n ∈ f` such that `s n ×ˢ s n ⊆ U`. -/
-def set_seq (n : ℕ) : set α :=  ⋂ m ∈ Iic n, (set_seq_aux hf U_mem m).val
+def set_seq (n : ℕ) : set α :=  ⋂ m ∈ set.Iic n, (set_seq_aux hf U_mem m).val
```

Comment:
Was that required?

Reply:
Yes. There is also a `finset.Iic` and changing imports put it in scope here.

Reply:
Ah, I missed the `open finset`.